tests/irq/main.c

```diff
@@ -53,6 +57,41 @@ int main(void)
 {
     puts("START");
 
+    /* check if we can control interrupt enabling properly */
+
+    /* initially, interrupts should be enabled */
+    if (!irq_is_enabled()) {
+        goto err;
```

Comment:
I'd prefer something like

```C
static inline int print_error(void)
{
    puts("FAILED");
    return 0;
}
```

and than replace `goto err;` by `return print_error();`. It would still be a one-line, but without the ["go to hell"-achievement](https://blog.gouline.net/achievement-unlocked-visual-studio-user-92ac27cfa6e4) unlocked

Reply:
doable and makes sense. Though will not touch this until the feasibility of this PR in general is proven.